modules/gdscript/doc_classes/@GDScript.xml

```diff
@@ -974,7 +974,7 @@
 			<return type="Array">
 			</return>
 			<description>
-				Returns an array with the given range. Range can be 1 argument N (0 to N-1), two arguments (initial, final-1) or three arguments (initial, final-1, increment).
+				Returns an array with the given range. [code]range()[/code] can have 1 argument N ([code]0[/code] to [code]N - 1[/code]), two arguments ([code]initial[/code], [code]final - 1[/code]) or three arguments ([code]initial[/code], [code]final - 1[/code], [code]increment[/code]). [code]increment[/code] can be negative. If [code]increment[/code] is negative, [code]final - 1[/code] will become [code]final + 1[/code]. Also, the initial value must be greater than the final value for the loop to run.
```

Comment:
It's actually `final - increment`. So if `increment` is -1, it's `final + 1`, but if it's -13, it will be `final + 13`. (Or maybe not, actually it might be `final - (final - initial) % increment` or something like that... Which shows that equations are maybe not the best way to convey how the `range()` arguments are used.

Reply:
I used the old wording again, so this should be resolved now.